addons/website/static/src/scss/website.scss

```diff
@@ -36,7 +36,33 @@ $-seen-urls: ();
     @include print-variable('has-customized-13-0-color-system', $o-has-customized-13-0-color-system);
 
     // 1) Handle default values
+    @include print-variable('text-margin-top', $text-margin-top);
+    @include print-variable('text-margin-bottom', $text-margin-top);
     @include print-variable('header-font-size', $font-size-base);
+    @include print-variable('h1-font-size', $h1-font-size);
+    @include print-variable('h2-font-size', $h2-font-size);
+    @include print-variable('h3-font-size', $h3-font-size);
+    @include print-variable('h4-font-size', $h4-font-size);
+    @include print-variable('h5-font-size', $h5-font-size);
+    @include print-variable('h6-font-size', $h6-font-size);
+    @include print-variable('headings-line-height', $headings-line-height);
+    @include print-variable('h2-line-height', $h2-line-height);
+    @include print-variable('h3-line-height', $h3-line-height);
+    @include print-variable('h4-line-height', $h4-line-height);
+    @include print-variable('h5-line-height', $h5-line-height);
+    @include print-variable('h6-line-height', $h6-line-height);
+    @include print-variable('headings-margin-top', $headings-margin-top);
+    @include print-variable('h2-margin-top', $h2-margin-top);
+    @include print-variable('h3-margin-top', $h3-margin-top);
+    @include print-variable('h4-margin-top', $h4-margin-top);
+    @include print-variable('h5-margin-top', $h5-margin-top);
+    @include print-variable('h6-margin-top', $h6-margin-top);
+    @include print-variable('headings-margin-bottom', $headings-margin-bottom);
+    @include print-variable('h2-margin-bottom', $h2-margin-bottom);
+    @include print-variable('h3-margin-bottom', $h3-margin-bottom);
+    @include print-variable('h4-margin-bottom', $h4-margin-bottom);
+    @include print-variable('h5-margin-bottom', $h5-margin-bottom);
+    @include print-variable('h6-margin-bottom', $h6-margin-bottom);
```

Comment:
Like the comment above says: those in the `// 1)` section should only handle default values. Here you force the printing of the real value, so it should probably be in `// 5)` (or use the default value as indicated which is currently the h1 config... but that won't be possible if we change to the h(x-1) config as suggested in my other comment).